addons/web/static/tests/views/fields/progress_bar_field_tests.js

```diff
@@ -650,4 +650,27 @@ QUnit.module("Fields", (hooks) => {
             assert.verifySteps(["Show error message"], "The error message was shown correctly");
         }
     );
+
+    QUnit.test(
+        "ProgressBarField: check the input value is valid or not in progressbar field",
```

Comment:
This message doesn't make the purpose of the test very clear. What do you think of something like : 
"ProgressBarField: writing overflowing int throws warning"?